src/components/GPUDashboardCardGraphs.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import {
+  ChartArea,
+  ChartDonutUtilization,
+  ChartGroup,
+  ChartVoronoiContainer,
+} from '@patternfly/react-charts';
+import { CardBody, CardFooter } from '@patternfly/react-core';
+import { humanize } from '@openshift-console/dynamic-plugin-sdk';
+import { GPUDashboardCardGraphsProps } from './types';
+
+import './GPUDashboardCardGraphs.css';
+
+export const GPUDashboardCardGraphs: React.FC<GPUDashboardCardGraphsProps> = ({
+  title,
+  ariaTitle,
+  rangeTitle,
+  rangeDescription,
+  scalarValue,
+  timeSerie,
+  unit,
```

Comment:
can we reuse `humanizeWatts`/`humanizePercentage`/[add other humanize functions](https://github.com/rh-ecosystem-edge/console-plugin-nvidia-gpu/blob/main/src/utils/cluster-overview.ts#L21-L40) instead of just using `unit` ?